manual.typ

```diff
@@ -54,7 +54,7 @@ module imported into the namespace.
 = Plot
 
 #doc-style.parse-show-module("/src/plot.typ")
-#for m in ("line", "bar", "boxwhisker", "contour", "errorbar", "annotation", "formats") {
+#for m in ("line", "bar", "boxwhisker", "contour", "errorbar", "annotation",) {
```

Comment:
Absolutely, I'll work on that tomorrow I think

Reply:
Ok so I'm looking at it and the plot documentation sufficiently describes relevant `axes` functions that it would be verbose to add them twice. What I'd suggest is that we keep it hidden (or only show it in some sort of "advanced" documentation) because my gut feeling is that it would confuse people.

One nitpick I have with myself is that tick formatters do belong in `axes` but they are only ever used as an argument for `{n}-format` in `plot.plot` - could we alias the tick formatters so they can be accessed through both `plot` and `axes`? Or would that set a bad precedent?